auto-complete.js

```diff
@@ -146,7 +148,7 @@
             }, that.sc);
 
             live('autocomplete-suggestion', 'mousedown', function (e) {
-                if (hasClass(e.target, 'autocomplete-suggestion')) { // else outside click
+                if (getClosest(e.target, '.autocomplete-suggestion')) { // else outside click
```

Comment:
This will trigger the autocomplete event even when the user clicks the `Remover` button. 
Here a suggestion to fix it:
```suggestion
                if (getClosest(e.target, '.autocomplete-suggestion') && !getClosest(e.target,'.autocomplete-suggestion--local-remove-button')) { // else outside click
```

Reply:
nice!